src/main/java/org/jitsi/videobridge/AbstractEndpoint.java

```diff
@@ -409,38 +409,44 @@ public abstract void setSenderVideoConstraints(
      * needs to receive from this endpoint
      */
     protected abstract void
-    maxReceiverVideoConstraintsChanged(VideoConstraints maxVideoConstraints);
+    maxReceiverVideoConstraintsChanged(@NotNull VideoConstraints maxVideoConstraints);
 
     /**
-     * Notifies this instance that the specified endpoint wants to receive
+     * Notifies this instance that a specified received wants to receive
      * the specified video constraints from the endpoint attached to this
      * instance (the sender).
      *
-     * @param endpointId the id that specifies the receiver endpoint
+     * The receiver can be either another endpoint, or a remote bridge.
+     *
+     * @param receiver the id that specifies the receiver endpoint
      * @param newVideoConstraints the video constraints that the receiver
      * wishes to receive.
      */
-    public void addReceiver(String endpointId, VideoConstraints newVideoConstraints)
+    public void addReceiver(String receiver, VideoConstraints newVideoConstraints)
```

Comment:
~`receivedEpId`?~ Nevermind, I see this can now refer to a relay ID as well. `receiverId`, maybe?